### PR TITLE
Fixes #17602 - mention-bot won't ping inactive past contributers

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: .mention-bot [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "requiredOrgs": ["theforeman"]
```
Comment (timogoebel) [inline .mention-bot]: I think, this is missing a trailing comma to be valid json.

File: .mention-bot [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+  "userBlackList": ["abenari", "unorthodoxgeek", "alongoldboim", "bgupta", "logicminds"]
```
Comment (timogoebel) [inline .mention-bot]: Shouldn't we "remove" this guys from the organization if they're definitely inactive?

**Review by timogoebel — CHANGES_REQUESTED** [review]
@tbrisker : Thanks. In addition to my inline comments, please file a redmine ticket for this.

File: .mention-bot (hunk 2 above)
Comment (tbrisker) [reply .mention-bot]: I think we should, at least from the committer group, but only @domcleal and @ohadlevy have the permissions to do that.

**Comment by tbrisker** [comment]
@timogoebel added ticket and fixed comment.

**Comment by dLobatog** [comment]
There's https://github.com/mention-bot/how-to-unsubscribe too..

**Comment by tbrisker** [comment]
@dLobatog thanks, that is useful for users not seeing notifications they don't care about but it will still ping them instead of relevant contributes, plus this is not useful for users that may be using mentionbot in other projects.

**Review by timogoebel — APPROVED** [review]
@tbrisker : Sorry for the delay, I was off. Looks good to me, will merge in a sec.

**Pull request merged**